src/ui/public/vislib/lib/types/point_series.js

```diff
@@ -4,15 +4,17 @@ export default function ColumnHandler(Private) {
 
   const createSeries = (cfg, series) => {
     const stacked = ['stacked', 'percentage', 'wiggle', 'silhouette'].includes(cfg.mode);
+    let interpolate = cfg.interpolate;
+    if (cfg.smoothLines) interpolate = 'cardinal';
```

Comment:
I'd add a comment here that this is for backward compatibility when loading URLs or configs.